src/entities/transformers/default-boolean.transformer.ts

```diff
@@ -0,0 +1,13 @@
+import { ValueTransformer } from 'typeorm';
+
+export class DefaultBooleanTransformer implements ValueTransformer {
+  constructor(private readonly defaultValue: boolean) {}
```

Comment:
```suggestion
  constructor(private readonly defaultValue: { from?: boolean, to?: number }) {}
```
이렇게 구성하는건 어떨까요??
현재 문제가 되는 부분이 isAllowComment와 isAnonymous여서 현재 DB 구조상 두 필드가 nullable하지 않아서 무조건 값이 있다고 생각해 그에따른 스펙에 맞춘거같은 인터페이스입니다.
boolean transform에 기본값을 허용한다는 기능에만 집중했을 때 양쪽에 필요할듯합니다.
그리고 현재 로직 상 from에 value가 없을 경우(undefined) false가 반환되어 DB default value 트리거가 동작하지 않을 가능성도 존재할거같아요. 이런 케이스도 { from: undefined }로 디폴트를 제공하면 해결되고요

Reply:
사실 from에 별 다른 조치가 필요한 이유을 잘 못느꼈습니다.
일단 제가 고민 해봤던 부분은

1. 엔티티 짤 때 tinyint(boolean) 컬럼에 nullable한 컬럼으로 만들 일이 없을 것 같다. db 내에서 값이 0일때와 null일 때의 기능적인 차이가 유의미해야 하는데 그런 상황 혹은 이유가 있나?
2. 1번의 이유에서 from은 별 다른 처리가 필요 없을 것 같다는 생각
    1. 1 —> true, 0 —> false
    2. from에서 value에 undefined가 들어올 가능성은 아예 없다고 생각
    3. 만약 nullable한 컬럼이라면 null을 그대로 return 하는 로직 추가

해당 하는 이유들 때문인데 석호님 말씀 하신걸 제가 잘 이해를 못한걸 수도 있습니다 ㅠㅠ

Reply:
네 1번 케이스를 생각하면 조치하지 않아도될거같네요